lib/fog/vsphere/requests/compute/get_storage_pod.rb

```diff
@@ -0,0 +1,30 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Real
+        def get_storage_pod(name, datacenter_name)
+          storage_pod = get_raw_storage_pod(name, datacenter_name)
+          raise(Fog::Compute::Vsphere::NotFound) unless storage_pod
+          storage_pod_attributes(storage_pod, datacenter_name)
+        end
+
+        protected
+
+        def get_raw_storage_pod(name, datacenter_name)
+          dc = find_raw_datacenter(datacenter_name)
+
+          @connection.serviceContent.viewManager.CreateContainerView({
+            :container  => dc,
+            :type       => ["StoragePod"],
+            :recursive  => true
+          }).view.select{|pod| pod.name == name}.first
+        end
+      end
+
+      class Mock
+        def get_storage_pod(name, datacenter_name)
+        end
```

Comment:
If you are not going to implement it i would say is better to not even define it since `fog` will define it on the fly with a custom exception telling it is not implemented.